addons/mrp/models/mrp_production.py

```diff
@@ -38,7 +38,7 @@ def _get_default_date_planned_start(self):
     def _get_default_is_locked(self):
         return not self.user_has_groups('mrp.group_unlocked_by_default')
 
-    name = fields.Char('Reference', copy=False, readonly=True)
+    name = fields.Char('Reference', default='New MO', copy=False, readonly=True)
```

Comment:
Keep "New" with the model name on top like this 
![image](https://user-images.githubusercontent.com/12071695/211039090-23205b45-3944-4c43-95d7-826c2680bdc1.png)